scripts/drupal.sh

```diff
@@ -82,13 +82,19 @@ $DRUSH_CMD -y en devel
 $DRUSH_CMD -y pm-uninstall search
 $DRUSH_CMD en -y search_api
 
-# Set default theme to bootstrap
-$DRUSH_CMD -y en bootstrap
-$DRUSH_CMD -y config-set system.theme default bootstrap
-## THIS IS STUPID
-# blocks are tied to themes because themes define the regions available to place blocks. gross.
-##
-
+# Set default theme to carapace (and download dependencies, will composer-ize later)
+$DRUSH_CMD --pm-force dl -y adaptivetheme
+$DRUSH_CMD --pm-force dl -y at_tools
+$DRUSH_CMD --pm-force dl -y layout_plugin
```

Comment:
The `drush --pm-force dl` calls have been replaced with `composer require` in https://github.com/Islandora-CLAW/claw_vagrant/pull/48/commits/532c02c83218f347aeba1010f1ae7ca666444f96